File: lab13/01.py
ls = []

while True:
    x = input()

    if x == '':
        break

    ls.append(x)

def create_factors_3_7(n):
    ls = []

    i = 1
    while len(ls) != n:
        if i % 3 == 0 or i % 7 == 0:
            ls.append(i)
        i += 1

    return ls

print(create_factors_3_7(21))

File: 113/lab5/01.c
#include <stdio.h>
#include <math.h>

int is_automorphic(int n1)
{
    long long n2 = powl(n1, 2);

    while (n1 > 0)
    {
        if (n1 % 10 != n2 % 10)
        {
            return 0;
        }

        n1 /= 10;
        n2 /= 10;
    }

    return 1;
}

int main()
{
    int n = 0;

    printf("Input n = ");
    scanf("%d", &n);

    long long npown = powl(n, 2);

    printf("n^2 = %lld\n", npown);

    if (is_automorphic(n))
    {
        printf("Yes. %d is automorphic number.\n", n);
    }
    else
    {
        printf("No. %d is not automorphic number.\n", n);
    }

    return 0;
}